pandas/io/gbq.py

```diff
@@ -230,40 +291,129 @@ def load_data(self, dataframe, dataset_id, table_id, chunksize, verbose):
 
         for index, row in dataframe.reset_index(drop=True).iterrows():
             row_dict = dict()
-            row_dict['json'] = json.loads(row.to_json(force_ascii = False,
-                                                      date_unit = 's',
-                                                      date_format = 'iso'))
+            row_dict['json'] = json.loads(row.to_json(force_ascii=False,
+                                                      date_unit='s',
+                                                      date_format='iso'))
             row_dict['insertId'] = job_id + str(index)
             rows.append(row_dict)
             remaining_rows -= 1
 
             if (len(rows) % chunksize == 0) or (remaining_rows == 0):
                 if verbose:
-                    sys.stdout.write("\rStreaming Insert is {0}% Complete".format(((total_rows - remaining_rows) * 100) / total_rows))
+                    sys.stdout.write("\rStreaming Insert is {0}% Complete".format(
+                        ((total_rows - remaining_rows) * 100) / total_rows))
                     sys.stdout.flush()
 
                 body = {'rows': rows}
-                response = self.service.tabledata().insertAll(
-                                                        projectId = self.project_id,
-                                                        datasetId = dataset_id,
-                                                        tableId = table_id,
-                                                        body = body).execute()
+                response = self.service.tabledata().insertAll(projectId=self.project_id, datasetId=dataset_id,
```

Comment:
what you changed is much more readable. When you have lots of parameters then should be listed individually